statsmodels/graphics/tsaplots.py

```diff
@@ -201,10 +201,7 @@ def seasonal_plot(grouped_x, xticklabels, ylabel=None, ax=None):
     ticks = []
     for season, df in grouped_x:
         df = df.copy() # or sort balks for series. may be better way
-        if pandas.__version__ < '0.17.0':
-            df.sort()
-        else:
-            df.sort_values(inplace=True)
+        df.sort(inplace=True)
```

Comment:
In old pandas this is a TimeSeries.  TimeSeries doesn't support inplace sorting, so just do

```
df = sorted_values(df)
```